internal/x/events/events.go

```diff
@@ -9,27 +9,26 @@ import (
 	"github.com/ory/x/otelx/semconv"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/trace"
-
-	"github.com/ory/keto/internal/x"
 )
 
-type Event string
-
 const (
-	RelationtuplesCreated Event = "RelationtuplesCreated"
-	RelationtuplesDeleted Event = "RelationtuplesDeleted"
-	RelationtuplesChanged Event = "RelationtuplesChanged"
+	EventRelationtuplesCreated semconv.Event = "RelationtuplesCreated"
```

Comment:
The `Event` prefix is not needed here IMHO because the package is already named `event`, so it will be always referenced as `events.EventRelationtuplesCreated`.

Reply:
True. I wanted to stick to the pattern that was established in ory/x, but this seems really superfluous.